modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== On iOS Safari, Hangul (Korean) characters will no longer merge onto the previous line after inserting a newline by pressing Enter.
+//#TINY-9746
+
+In previous versions of {productname}, in iOS Safari, a bug has been fixed regarding the handling of Korean characters.
+
+Previously, when a newline was inserted by pressing the Enter key, Korean characters would merge onto the previous line instead of starting a new line. This occurred because iOS Safari composed Korean characters by deleting the previous partial character and inserting the composed character, continuing until the Enter key was pressed. Even though {productname} has its own Enter key handler, the native Enter key event was suppressed, causing the bug.
```

Comment:
```suggestion
Previously, when a newline was inserted by pressing the Enter key and the user continued typing in Korean without moving the caret, Korean characters typed on the newline would merge onto the previous line and the newline would be removed. This occurred because iOS Safari composed Korean characters by deleting the previous partial character and inserting the composed character, continuing until a native Enter `keypress` event had been fired. However, {productname} has its own Enter key handler on `keydown`, so the native Enter `keypress` event was suppressed, preventing iOS Safari from stopping Korean character composition as needed and causing the bug.
```